packages/core/src/config/sanityConfig.ts

```diff
@@ -63,6 +63,17 @@ export interface DocumentHandle<
   documentId: string
 }
 
+/**
+ * Identifies a specific asset within a Sanity project/dataset.
+ * Includes optional `projectId`/`dataset` and the required `assetId` (asset document `_id`).
+ * Useful for cross-instance asset operations without relying on the active ResourceProvider.
```

Comment:
I don't think I fully understand this comment -- what is meant by cross-instance? I'm also a little wary of the mental model of "this is a document but the _id has a different parameter name"